data/json/recipes/recipe_armor.json

```diff
@@ -8339,17 +8339,63 @@
       [ "hat_hunting", 1 ]
     ],
     [
+      [ "mask_dust", 1 ],
```

Comment:
Ok, I think I'll use the dust mask in the scavenger cowl recipe then, instead of filter mask (since dust masks are non-craftable).